changelog/0015-invoke-celery-directly.yml

```diff
@@ -0,0 +1,17 @@
+title: Update supervisor confs to invoke celery directly
+key: invoke-celery-directly
+date: 2019-02-22
+optional_per_env: no
+min_commcare_version: f67f9c8149bd71712f457c270b03b5a55077c273
+max_commcare_version:
```

Comment:
```max_commcare_version``` will be added retroactively after https://github.com/dimagi/commcare-hq/pull/21588 is merged.